Dockerfile

```diff
@@ -21,10 +21,13 @@ MAINTAINER Evan Brown <evanbrown@google.com>
 
 RUN rm /etc/nginx/conf.d/*.conf
 
+RUN mkdir -p /etc/nginx/extra-conf.d
```

Comment:
I am not really sure, what you'd expect. I am not using packer myself (yet). Something like:

```
{
      "type": "shell",
      "inline": [
        "rm /etc/nginx/conf.d/*.conf",
        "mkdir -p /etc/nginx/extra-conf.d"
      ]
    }
```
Separate PR? Or this PR, don't force me to squash my commits. :)